### ISSUE TITLE
NoMoreDWM does not work,

### ISSUE THREAD

**Issue by mantanaNord** [TK50P/NoMoreDWM#4]
does not work, placed on the desktop, launched as administrator and without, after launch and agreement with all points, the system restarts, after restarting dwm continues to work. build 19045.3803, version 22h2

**Comment by TK50P** [comment]
Have you waited for 5 seconds? because disable/enable script needs 5 seconds to apply changes.

**Comment by mantanaNord** [comment]
of course, I reboot only after notification from the application itself, now I tried to wait more than one minute, unsuccessfully

**Comment by TK50P** [comment]
Also Have you tried to update to Windows as latest version and tried? I have not tried but I will try this on VM.
